spark-bigquery-pushdown/pushdown_common_src/test/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverterSuite.scala

```diff
@@ -797,6 +797,13 @@ class SparkExpressionConverterSuite extends AnyFunSuite with BeforeAndAfter {
     assert(bigQuerySQLStatement.get.toString == "CAST ( SCHOOLID AS FLOAT64 )")
   }
 
+  test("convertMiscellaneousExpressions with Cast from Integer to Bytes") {
```

Comment:
Might as well add unit tests for cast from the other types to bytes :)

Reply:
Added